doc/whats-new.rst

```diff
@@ -14,11 +14,24 @@ What's New
 
     np.random.seed(123456)
 
-.. _whats-new.0.17.1:
+.. _whats-new.0.18.0:
 
-v0.17.1 (unreleased)
+v0.18.0 (6 May 2021)
 ---------------------
```

Comment:
you'll have to check, but this has to be exactly as long as the line directly above. I think that means there is one `-` you have to remove?
```suggestion
--------------------
```